test/collection/test_config.py

```diff
@@ -811,6 +811,40 @@ def test_config_with_vectorizer_and_properties(
             }
         },
     ),
+    (
+        Configure.Generative.databricks(),
+        {"generative-databricks": {}},
+    ),
+    (
+        Configure.Generative.databricks(
+            model="model",
+            base_url="https://api.databricks.com",
```

Comment:
is `base_url` referring to the `endpoint` setting of the generative-databricks module?

Reply:
This is definitely a artefact from copying from another config, will remove! Thanks for the catch 😁